packages/odyssey-storybook/src/components/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -45,6 +46,7 @@ export default {
     required: { control: "boolean" },
     value: { control: "text" },
   },
+  decorators: [MuiThemeDecorator],
```

Comment:
We should be able to remove the theme wrappers from this component now, since they're in the decorator

Reply:
Are they still necessary for the Custom theme example included below? It's not a normal use case, but I believe @jeffbelser-okta was using this to show how components could be themed on top of our default.